### PR TITLE
HPCC-25950 Institute lower default log level

### PR DESCRIPTION
**Pull request by rpastrana**
- Updates global default log level
- Updates helm chart global default log level

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexisrisk.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by jakesmith — COMMENTED** [review]
@rpastrana - changes look good, but there some other places that currently also have a default of 100, which should be changed at same time.
e.g. the defaultYaml in roxie, daserver and saserver.

I think we should just remove those override setting, to match others, and to pick up global default.

**Comment by rpastrana** [comment]
@jakesmith please review.

**Comment by rpastrana** [comment]
@AttilaVamos the check failure seems unrelated:
"CMake Error at cmake_modules/commonSetup.cmake:756 (MESSAGE):
-- Found Bison v3.5.1
  You need Gnu c++ version 7.3.0 or later to build this project (version
-- Configuring incomplete, errors occurred!"

**Comment by mckellyln** [comment]
The error "You need Gnu c++ version 7.3.0 or later to build this project ..." occurs sometimes on additional commits or a squash.  It is something to do with re-running cmake and its cache - it is not related to anything specific to this PR.
@jakesmith has looked into it and I have as well but it is difficult to reproduce.
@AttilaVamos fyi

File: dali/dfu/dfuserver.cpp [hunk 1]
```diff
@@ -104,6 +104,8 @@ version: "1.0"
   name: dfuserver
   monitorInterval: 900
   maxJobs: 1
+  logging:
+    detail: 100
```
Comment (jakesmith) [inline dali/dfu/dfuserver.cpp:108]: not sure why this was added?

**Review by jakesmith — CHANGES_REQUESTED** [review]
@rpastrana - There's a couple of other components that override the default (used to be same as default) unnecessarily, that should also be removed.

**Comment by rpastrana** [comment]
> 
> 
> @rpastrana - There's a couple of other components that override the default (used to be same as default) unnecessarily, that should also be removed.

I didn't feel comfortable overriding those components' explicit logging levels because there is evidence (at least in bare metal) they are meant to log at the highest level. 
Looks like you @jakesmith were involved in the sasha baremetal and container logging detail:
https://github.com/hpcc-systems/HPCC-Platform/blame/b895abd1e8d54ae9d6d4a3675037d405b576e4a1/dali/sasha/saserver.cpp#L357
https://github.com/hpcc-systems/HPCC-Platform/blob/b895abd1e8d54ae9d6d4a3675037d405b576e4a1/dali/sasha/saserver.cpp#L357
If you confirm it should default, I'll go ahead and remove.

@richardkchapman can you confirm this Roxie config entry can be removed and allowed to default?
https://github.com/hpcc-systems/HPCC-Platform/blob/b4bdebe56a7c40af5631559bb5d9b9581a5604ca/roxie/roxie/roxie.cpp#L55

**Comment by jakesmith** [comment]
> > @rpastrana - There's a couple of other components that override the default (used to be same as default) unnecessarily, that should also be removed.
> 
> I didn't feel comfortable overriding those components' explicit logging levels because there is evidence (at least in bare metal) they are meant to log at the highest level.
> Looks like you @jakesmith were involved in the sasha baremetal and container logging detail:
> https://github.com/hpcc-systems/HPCC-Platform/blame/b895abd1e8d54ae9d6d4a3675037d405b576e4a1/dali/sasha/saserver.cpp#L357
> https://github.com/hpcc-systems/HPCC-Platform/blob/b895abd1e8d54ae9d6d4a3675037d405b576e4a1/dali/sasha/saserver.cpp#L357
> 
> 
> If you confirm it should default, I'll go ahead and remove.
> @richardkchapman can you confirm this Roxie config entry can be removed and allowed to default?
> https://github.com/hpcc-systems/HPCC-Platform/blob/b4bdebe56a7c40af5631559bb5d9b9581a5604ca/roxie/roxie/roxie.cpp#L55

@rpastrana - please go ahead and remove the sasha one.

Before this PR, these per component overrides (of detail: 100) were effectively no-op's, setting the component default to what the global default already specified. I don't think they were intentional, just there as a sort of self-documenting attribute.
Sasha doesn't want to raise the global logging default above the global logging default.
I don't think anywhere does.

**Comment by rpastrana** [comment]
@jakesmith updated per your suggestions.

**Review by jakesmith — APPROVED** [review]
@rpastrana - looks good.

**Pull request merged**